src/coreclr/src/vm/clsload.hpp

```diff
@@ -301,6 +301,8 @@ class NameHandle
 
 //-------------------------------------------------------------------------------------------
 //
+// TODO: Clean this up
```

Comment:
Is there an issue tracking this?

Reply:
I am going to clean it up as part of this PR. I submitted this as draft to see whether there are any tests failing.